src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java

```diff
@@ -91,6 +91,10 @@ private static boolean isTlsEnabled(String target, AuthAndTLSOptions options) {
     if (options.incompatibleTlsEnabledRemoved && options.tlsEnabled) {
       throw new IllegalArgumentException("flag --tls_enabled was not found");
     }
+    if (options.incompatibleTlsEnabledRemoved) {
+      // 'grpcs://' or empty prefix => TLS-enabled
```

Comment:
Maybe expand the comment on empty prefix a bit?

Reply:
done